### PR TITLE
Add Sushiswap trades on BNB chain and Polygon

### PR DESCRIPTION
**Pull request by chrispearcx**
Brief comments on the purpose of your changes:

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Comment by jeff-dude** [comment]
hi @Hosuke @chrispearcx please continue on this PR, i've closed the other one due to inactivity

**Comment by Hosuke** [comment]
I will try to recycle some seeds from #2000 for testing.

File: seeds/dex/trades/dex_trades_seed.csv [hunk 1]
```diff
@@ -337,3 +337,17 @@ bnb,onepunchswap,normal,2023-01-12,0xa723fca3a321586a38c738a1bc221a4e9bf4f5e7732
...
+polygon,sushiswap,1,2021-06-16 0:00,0xa727f3165a6e7533cbeecfbe67e4072f8b9a0eec486a4c9976b25b902a1bd6f0,310,0x7ceb23fd6bc0add59e62ac25578270cff1b9f619,0.0008197673253,0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270,1.388941988
+
+bnb,sushiswap,1,2022-11-22,0x683b4299466b4fec3fac2859e2dd27215f69879b31a7384593b46aef71952ed5,373,0xa0ed3c520dc0632657ad2eaaf19e26c4fd431a84,6571.818499746365907562,0xe9e7cea3dedca5984780bafc599bd69add087d56,20
+bnb,sushiswap,1,2022-11-22,0xa572f3ea4bcc491c3d6a3b54f2cd59c776238c3187c1ee7975d9635f8a55d129,260,0xe9e7cea3dedca5984780bafc599bd69add087d56,30.330984668706979348,0xa0ed3c520dc0632657ad2eaaf19e26c4fd431a84,10000
+bnb,sushiswap,1,2022-11-22,0xfac7881f1461c9c56a11f68647385ea27c4d66849f646d5f377e558014652df2,536,0xe9e7cea3dedca5984780bafc599bd69add087d56,0.003061259047161221,0xa0ed3c520dc0632657ad2eaaf19e26c4fd431a84,1
+bnb,sushiswap,1,2023-01-17,0x5169181339080e2d56c13c0a075dac19f19e3f8300a8c068e8f4519bf424e7ab,186,0xe9e7cea3dedca5984780bafc599bd69add087d56,2.487085655648097925,0xc598275452fa319d75ee5f176fd3b8384925b425,177.678
```
Comment (Hosuke) [inline seeds/dex/trades/dex_trades_seed.csv:348]: Edge case 1:
https://bscscan.com/tx/0x5169181339080e2d56c13c0a075dac19f19e3f8300a8c068e8f4519bf424e7ab
This seed is currently not covered by the model.

Comment (Hosuke) [reply seeds/dex/trades/dex_trades_seed.csv]: Sorry, the `ext_index` for this seed record should be `166` instead of `186`

**Comment by Hosuke** [comment]
Seems this PR passes all final seeds in #2000. I will remove the extra seeds in the next commit.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @chrispearcx 

- Metadata & naming conventions ✅
- Config blocks match uniqueness tests in schema files ✅
- Tests and seeds added for the new models ✅
- Incremental filters added when joining large tables ✅
- Contract addresses are lowercase ✅
- Inner join to large tables(*.transactions) as possible ✅
- `CAST()` used in the base models ✅

[Sample sushiswap_bnb_trades]()
[Sample sushiswap_polygon_trades]()

**Review by jeff-dude — APPROVED** [review]
thank you @chrispearcx for the contribution and @Hosuke for pushing it over the finish line!

**Pull request merged**